src/components/shared/LibraryTypeSelect.vue

```diff
@@ -71,7 +71,7 @@ export default {
         return []
       }
     },
-    nullOption: ({ allowNone }) => (allowNone ? [{ value: null, text: 'None' }] : []),
+    nullOption: ({ allowNone }) => (allowNone ? [{ value: '', text: 'None' }] : []),
```

Comment:
maybe rename to emptyOption as this no longer returns null

Reply:
👍